packages/grafana-runtime/src/config.ts

```diff
@@ -0,0 +1,78 @@
+import extend from 'lodash/extend';
+import { GrafanaTheme, getTheme, GrafanaThemeType, PanelPluginMeta, DataSourceInstanceSettings } from '@grafana/ui';
+
+export interface BuildInfo {
+  version: string;
+  commit: string;
+  isEnterprise: boolean;
+  env: string;
+  latestVersion: string;
+  hasUpdate: boolean;
+}
+
+export class Settings {
```

Comment:
Would be good to take the time to change this type name to something config (as the file is called config). Maybe AppBootConfig or just GrafanaAppConfig or along that vein.

Reply:
I renamed it GrafanaBootConfig :)